portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+function addRandomLanguage() {
+  const languages =
+      ['Español', 'English', 'Deutsch'];
+ 
+  // Pick a random language.
+  const language = languages[Math.floor(Math.random() * languages.length)];
+ 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+  const languageContainer = document.getElementById('language-container');
+  languageContainer.innerText = language;
+}
```

Comment:
Make sure you have a newline at the end of your file - see https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file for context! Same elsewhere!

Reply:
Ack!